### ISSUE TITLE
/FI compiler option isn't supported

### ISSUE THREAD

**Issue by kelleyma49** [inorton/cclash#18]
/FI compiler option isn't a supported compiler option.

**Comment by inorton** [comment]
Do you mean?

`/FI<file> name forced include file`

I don't recall implementing that one. I would have thought though those would be noticed. Do you get an error? or false negatives? false positives?

**Comment by kelleyma49** [comment]
Yes.  see https://github.com/inorton/cclash/blob/master/CClash/Compiler.cs#L461; notice how `ProcessArguments()` returns false if the /FI compiler option is encountered.
